### PR TITLE
Fix dropzone

### PR DESCRIPTION
**Pull request by pash90**
Allows the upload of one or more files and also shows the preview of the file uploaded, if possible

### CODE REVIEW

File: src/DropZone/DropZone.jsx [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
-import {first} from "lodash";
-import DropZone from "react-dropzone";
-import classnames from "classnames";
-import {compose} from "recompose";
-import performanceWrapper from "../Form/Helpers/performanceWrapper";
-import FileItem from "./FileItem";
-import {getHTMLAttributes} from "../Form/Helpers/inputHelpers";
-import {DropZoneProps, DropZoneFile} from "../Form/Types/types";
-import {PerformanceWrapperProps} from "../Form/Helpers/performanceWrapper";
+/**
```
Comment (shane935) [inline src/DropZone/DropZone.jsx]: Can we remove these comments

File: src/DropZone/DropZone.jsx [hunk 2]
```diff
@@ -39,32 +54,39 @@ class DropZoneFrecl extends React.Component{
     const {value} = this.props;
     return isFileArray(value) ? value : isSingleFile(value) ? value : null;
   }
+  
   onDrop = (files) => {
-    if (this.props.multiple !== false) {
-      const immutFiles = List(files);
+    const {multiple, inputChanged} = this.props;
+    const droppedFiles = List(files).map(file => Map(file));
```
Comment (shane935) [inline src/DropZone/DropZone.jsx]: You can't convert files to an Immutable Map

File: src/DropZone/DropZone.jsx (hunk 1 above)
Comment (pash90) [inline src/DropZone/DropZone.jsx]: nope

**Comment by pash90** [comment]
fixes #53

**Comment by pash90** [comment]
@shane935 Apart from your review of removing comments, this PR was approved by you as a handshake deal. Merging it in

**Pull request merged**